src/app/config.rs

```diff
@@ -299,8 +299,8 @@ const DEFAULT_RESOLV_CONF: &str = "/etc/resolv.conf";
 
 /// It's assumed that a typical proxy can serve inbound traffic for up to 100 pod-local
```

Comment:
This comment is definitely out of date now and should be updated (or just removed).

Reply:
Does it make sense that inbound would really need 10,000 different authorities?

Reply:
I think it would be okay if inbound had a smaller capacity than outbound, but I didn't want to just assume "inbound can always be half of outbound" or something like that. We could even keep it at 100 probably. The reason for this change is really coming from the outbound side of things.

Reply:
@hawkw With adjusting inbound capacity back to 100, if my understanding is correct then this comment is still accurate.

Reply:
@kleimkuhler yup, let's leave it as-is 👍